File: LoginGUI/src/Shift.java
import java.util.*;
import java.text.*;


public class Shift {
	private int shiftID;
	private String shiftDate;
	private String shiftTime;
	private String shiftRequirements;
	
	public Shift(int shiftId, String shiftDate, String shiftTime, String shiftRequirements) {
		this.shiftID = shiftId;
		this.shiftDate = shiftDate;
		this.shiftTime = shiftTime;
		this.shiftRequirements = shiftRequirements;
	}

	
	//Getters and Setters
	public int getShiftID() {
		return shiftID;
	}

	public void setShiftID(int shiftID) {
		this.shiftID = shiftID;
	}

	public String getShiftDate() {
		return shiftDate;
	}

	public void setShiftDate(String shiftDate) {
		this.shiftDate = shiftDate;
	}

	public String getShiftTime() {
		return shiftTime;
	}

	public void setShiftTime(String shiftTime) {
		this.shiftTime = shiftTime;
	}

	public String getShiftRequirements() {
		return shiftRequirements;
	}

	public void setShiftRequirements(String shiftRequirements) {
		this.shiftRequirements = shiftRequirements;
	}
	
	public String toString() {
		return "" + shiftID + " " + shiftDate + " " + shiftTime + " " + shiftRequirements;
	}
}

File: DBConnectionTest/Models/Shift.cs
using System;
namespace DBConnectionTest.Models
{
    public class Shift
    {
       public Shift(){}
       public Shift(long v1, long v2, DateTime dateTime, int v3, string v4)
      {
         this.ShiftId = v1;
         this.CompanyId = v2;
         this.ShiftDate = dateTime;
         this.ShiftTime = v3;
         this.ShiftRequirements = v4;
      }
      public long ShiftId { get; set; }
        public long CompanyId { get; set; }
        public DateTime ShiftDate { get; set; }
        public int ShiftTime { get; set; }
        public string ShiftRequirements { get; set; }
      

    }
}